diff --git a/node_modules/@grpc/proto-loader/build/src/index.js b/node_modules/@grpc/proto-loader/build/src/index.js
index 9567584..7043899 100644
--- a/node_modules/@grpc/proto-loader/build/src/index.js
+++ b/node_modules/@grpc/proto-loader/build/src/index.js
@@ -96,3 +96,4 @@
                     break;
                 default:
                     throw new Error(`Invalid JSON representation of value for field ${field.fullName} of type ${field.type}`);
+            }}
\ No newline at end of file
